src/components/Avatar/index.tsx

```diff
@@ -60,27 +65,38 @@ const Avatar: FunctionComponent<AvatarProps> = ({
   image = avatar,
   size = 32,
   text,
-  textBgColor = 'lightViolet',
-  textColor = 'white',
+  textBgColor = 'primary',
+  textColor = 'primary',
   textSize = 10,
   lock = false,
   ...props
-}) => (
-  <Box width={size} height={size} position="relative" {...props}>
-    {text ? (
-      <StyledDiv
-        lock={lock}
-        textBgColor={textBgColor}
-        textColor={textColor}
-        textSize={textSize}
-      >
-        {lock ? <Icon name="lock" color="gray550" /> : formatTextToAvatar(text)}
-      </StyledDiv>
-    ) : (
-      <StyledImg src={image} alt="" />
-    )}
-  </Box>
-)
+}) => {
+  const theme = useTheme()
+
+  return (
+    <Box width={size} height={size} position="relative" {...props}>
+      {text ? (
+        <StyledDiv
+          lock={lock}
+          textBgColor={textBgColor}
+          textColor={textColor}
+          textSize={textSize}
+        >
+          {lock ? (
+            <Icon
+              name="lock"
+              color={transparentize(0.5, theme.colors.neutral.text)}
```

Comment:
Here you can directly use `theme.colors.neutral.textWeak` modification https://github.com/scaleway/scaleway-ui/pull/981/files will include correct shade of gray

Reply:
done